src/modules/timepix3/generators/odin_datagen.bash

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Parse command line arguments
+while getopts "f:d" opt; do
```

Comment:
This is pretty old school :-) Are you familiar with argparse?

import argparse

https://docs.python.org/3/library/argparse.html

It is a bit more flexible IMO